src/main/kotlin/com/evalify/evalifybackend/quiz/service/QuizService.kt

```
@@ -0,0 +1,7 @@
+package com.evalify.evalifybackend.quiz.service
+import com.evalify.evalifybackend.quiz.repository.QuizRepository
+import org.springframework.stereotype.Service
+
+@Service
+class QuizService(quizRepository: QuizRepository) {
+}
```

detekt (1.23.8)

[warning] The class or object QuizService is empty.

(detekt.empty-blocks.EmptyClassBlock)